research/conditional/utils/layer_manager.py

```diff
@@ -98,7 +98,7 @@ def update_cache_for_logging(self, key, value):
             else:
                 self.logging_cache[key] = value.clone().detach().cpu()
 
-    def _combine_to_dict_key(key, layer_type, block_number):
+    def _combine_to_dict_key(self, key, layer_type, block_number):
```

Comment:
It just didn't work before, right? Insane. Unittests would be nice i guess.

Reply:
yuh